pkgs/by-name/em/emacsclient-commands/package.nix

```
@@ -0,0 +1,28 @@
+{ pkgs, lib, stdenv, fetchFromGitHub, buildGoModule, runCommandLocal }:
+
+buildGoModule rec {
```

```suggestion
buildGoModule {
```
you can remove the rec if you hard code the repo name